src/Section.js

```diff
@@ -2,6 +2,9 @@ import React, { PropTypes } from 'react';
 import { marg, pad } from './styles';
 
 const Section = React.createClass({
+/**
+ * Section is a layout primitive.
+ */
```

Comment:
It is _not_ set later in the module either (like some of the others, e.g. `ShowMoreEllipsis`).